docs/source/install.rst

```diff
@@ -0,0 +1,232 @@
+***********************
+Building and Installing
+***********************
+
+Prerequisites
+=============
+
+-  A ROCm enabled platform, more information `here <https://rocm.github.io/>`_.
+
+Installing pre-built packages
+=============================
+
+Download pre-built packages either from `ROCm's package servers <https://rocm.github.io/install.html#installing-from-amd-rocm-repositories>`_ or by clicking the GitHub releases tab and manually downloading, which could be newer.  Release notes are available for each release on the releases tab.
+
+.. code-block::bash
+   sudo apt update && sudo apt install hipblas
+
+Quickstart hipBLAS build
+========================
+
+Build library dependencies + library
+------------------------------------
+The root of this repository has a helper bash script `install.sh` to build and install hipBLAS on Ubuntu with a single command.  It does take a lot of options and hard-codes configuration that can be specified through invoking cmake directly, but it's a great way to get started quickly and can serve as an example of how to build/install.
+A few commands in the script need sudo access so that it may prompt you for a password.
```

Comment:
AFAIK the script install.sh works for Ubuntu, Centos, Fedora, SLES, and Windows. The Ubuntu Comment is old. I think it would be better to replace "build and install hipBLAS on Ubuntu with a single command." with something like "build and install hipBLAS with a single command." It has been tested on the Linux distributions listed in install.sh."  . I am not sure if we are officially supporting Windows. We can add Windows when we officially support it.